src/main/kotlin/it/pagopa/wallet/controllers/WalletController.kt

```diff
@@ -43,13 +44,13 @@ class WalletController(
     private val webClient: WebClient = WebClient.create(),
 ) : WalletsApi {
 
+    @SuppressWarnings("kotlin:S6508")
```

Comment:
Please remove this suppressWarnings

Reply:
done here c862cd6bcc1b4ddb1bb866ee039c4a18bb557003

Reply:
I think u removed it from the wrong method

Reply:
done here 4288cda0b62d2bebd26b7f3dc045b6653acd84e2